v19.2/split-at.md

```diff
@@ -252,3 +252,4 @@ SHOW RANGES FROM TABLE t;
 - [Distribution Layer](architecture/distribution-layer.html)
 - [Replication Layer](architecture/replication-layer.html)
 - [`SHOW JOBS`](show-jobs.html)
+- [`UNSPLIT AT`](unsplit-at.html)
```

Comment:
Same comment as previously re: "new in 19.2" flag.  Also, I think the SPLIT AT page should be updated to link to the Range Merges doc and to note that you have to explicitly turn off Range Merges if you want to use SPLIT AT.  Perhaps that should be a separate issue?  UPDATE: I did a search and found that https://github.com/cockroachdb/docs/issues/4960 has been filed to address this.  I've added it to the 19.2 milestone as a P-1.